src/octopeer-github/main/Database/ConsoleLogDatabaseAdapter.ts

```diff
@@ -1,12 +1,31 @@
 /**
  * Created by Youri on 04/05/2016.
- */
-/**
  * Semi-mock class for being able to check implementation completeness.
  */
 class ConsoleLogDatabaseAdapter implements DatabaseAdaptable {
 
     /**
+     * Simple boolean to keep track of the raw data output.
+     */
+    private static outputRawData = true;
+
+    /**
+     * Set the raw data switch for the console.
+     * @bool the new value.
+     */
+    public static setRawData(bool: boolean) {
```

Comment:
Can you give the parameter a more descriptive name and a default value (then you don't need the type annotation anymore)? Example (also for the name) can be found in `Logger.setDebug()`.